packages/moonstone/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/Input.InputSpotlightDecorator` to add unLockedPointer config
```

Comment:
I think that I would be better to move this CHANGELOG from Fixed to Added with the format like the following changelog.

```
### Added

- `moonstone/Heading` prop `spacing` with default value `'small'`
```

Reply:
The change written in CHANGELOG is removed. (Roy mentioned the reason below)